.github/workflows/test-rust.yaml

```diff
@@ -86,7 +86,7 @@ jobs:
         working-directory: ./crates/prql-compiler/tests/integration
         if: ${{ inputs.target == 'x86_64-unknown-linux-gnu' }}
       - name: Wait for database
-        uses: ifaxity/wait-on-action@v1
+        uses: ifaxity/wait-on-action@v1.1.0
```

Comment:
Doesn't `v1` resolve to the latest?

Reply:
Not resolved.
This actually refers to a tag in the Git repository, and the v1 and v1.1.0 tags are tied to different commits because the v1 tag has not been updated.

Reply:
Ah interesting, TIL.

I wonder why dependabot didn't upgrade, in that case...

Reply:
Interestingly GH recommends that actions change their `v1` tag to the latest `v1` release: https://github.com/actions/toolkit/blob/main/docs/action-versioning.md...

Reply:
Good point.
I think Dependabot has judged `v1` is greater than `v1.1.0`...

Reply:
> I think Dependabot has judged `v1` is greater than `v1.1.0`...

😬

Reply:
Oh, and actually that's what the issue you linked says...